typing_tester.py

```diff
@@ -0,0 +1,137 @@
+import curses
+from curses import wrapper
+import time
+
+def cal_accuracy_score(given_word, user_text, no_of_backspaces):
+    """Calculates the accuracy score for typing tester"""
+
+    wrong_letter = 0
+    try:
+        wrong_letter = sum([1 for i in range(len(given_word)) if user_text[i] != given_word[i]])
+    except:
+        wrong_letter += 1
+
+    total_mistakes = wrong_letter + no_of_backspaces
+    accuracy = len(given_word) - total_mistakes
+    accuracy =  accuracy/len(given_word)*100
+
+    return float(f"{accuracy:.2f}")
+
+
+def cal_time_score(elapsed_time, given_word):
+    """Calculates the time score for typing tester"""
+
+    # Avg. time for one keystroke
+    keystroke_time = 0.28 
+    # Avg. time to hover hands from mouse to keyboard
+    mouse_to_keyboard_time = 0.4 
+    # formula to calculate raw score
+    raw_score = elapsed_time / (mouse_to_keyboard_time + (keystroke_time * (len(given_word) + 1)))
+
+    if raw_score < 1:
+         return float(100)
+    else:
+         return float(f"{(100/raw_score):.2f}")
```

Comment:
Make use of ternary operator to make it one liner

Reply:
Resolved